metadata.rb

```diff
@@ -9,10 +9,12 @@
 recipe 'samba::client', 'Installs smbclient package'
 recipe 'samba::server', 'Installs samba server packages and configures smb.conf'
 
-%w( debian ubuntu centos fedora redhat scientific amazon oracle ).each do |os|
```

Comment:
scientifc and oracle just come along for free with RHEL. We actually have a bento box for oracle too if you want to test it.